web/src/components/inputs/common.tsx

```diff
@@ -51,4 +51,45 @@ const CheckboxField = ({
   </div>
 );
 
-export { ErrorField, CheckboxField };
\ No newline at end of file
+interface CheckboxFieldIconProps {
+  name: string;
+  label: string;
+  sublabel?: string;
+  disabled?: boolean;
```

Comment:
Instead of making a new component that have the Icon added, you can add new props/properties to the existing one.

You could add this to the `CheckboxFieldProps` interface, which would then make it able to optionally take a tooltip in the existing component.

`tooltip?: JSX.Element;`